Sources/Views/Cells/AudioMessageCell.swift

```diff
@@ -28,32 +28,26 @@ import AVFoundation
 /// A subclass of `MessageContentCell` used to display video and audio messages.
 open class AudioMessageCell: MessageContentCell {
 
-    /// The `ImageName` enum holds the names of default iamges used to decorate play button
```

Comment:
These changes seem outside the scope of this PR

Reply:
Not quite, the `AudioMessageCell` expose a method to get a image resource with in framework by a given name. I had extracted this method in a separate extension `UIImage+Extension`. This way both classes `AudioMessageCell` and `ContactMessageCell` can easily access resources.